<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConFiguration</class>
 <widget class="QWidget" name="ConFiguration">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>420</width>
    <height>382</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_17">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_8">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>设备索引号</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="comboBox_device_id"/>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>18</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_7">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>选择CAN通道</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="comboBox_can">
             <item>
              <property name="text">
               <string>通道1</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>通道2</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QCheckBox" name="checkBox_all_open_can">
           <property name="text">
            <string>打开所有通道</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>CAN参数</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;波特率&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboBox_bps">
            <item>
             <property name="text">
              <string>Self Define</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>10k bps</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>20k bps</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>40k bps</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>50k bps</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>80k bps</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>100k bps</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>125k bps</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>200k bps</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>250k bps</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>400k bps</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>500k bps</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>666k bps</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>800k bps</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>1000k bps</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>33.33k bps</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>66.66k bps</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>83.33k bps</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QLabel" name="label_6">
              <property name="text">
               <string>滤波方式</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="comboBox_receive_type">
              <item>
               <property name="text">
                <string>接收所有类型</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>只接收标准帧</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>只接收扩展帧</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>过滤验收码</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_accode">
            <property name="text">
             <string>0x80000000</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>过滤屏蔽码</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_AccMask">
            <property name="text">
             <string>0xFFFFFFFF</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>工作模式</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboBox_work_type">
            <item>
             <property name="text">
              <string>正常工作</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>监听</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>自发自收</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>正常工作并接收错误帧</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>CAN发送</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_14">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_7">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_6">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_16">
              <item>
               <widget class="QLabel" name="label_12">
                <property name="text">
                 <string>发送总帧数</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="lineEdit_all_frames">
                <property name="text">
                 <string>48</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="checkBox_IDadd">
                <property name="text">
                 <string>ID增加</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_11">
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_12">
                <item>
                 <widget class="QLabel" name="label_8">
                  <property name="text">
                   <string>帧格式</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QComboBox" name="comboBox">
                  <item>
                   <property name="text">
                    <string>标准帧</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>扩展帧</string>
                   </property>
                  </item>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_13">
                <item>
                 <widget class="QLabel" name="label_9">
                  <property name="text">
                   <string>帧类型</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QComboBox" name="comboBox_2">
                  <item>
                   <property name="text">
                    <string>数据帧</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>远程帧</string>
                   </property>
                  </item>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_15">
                <item>
                 <widget class="QLabel" name="label_10">
                  <property name="text">
                   <string>帧ID</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="lineEdit_ID"/>
                </item>
               </layout>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_19">
            <item>
             <widget class="QLabel" name="label_14">
              <property name="text">
               <string>发送周期</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit_cycle"/>
            </item>
            <item>
             <widget class="QLabel" name="label_15">
              <property name="text">
               <string>ms</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="checkBox_num_add">
              <property name="text">
               <string>数据增加</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_10">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_9">
         <item>
          <widget class="QPushButton" name="pushButton_ok">
           <property name="text">
            <string>确定</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_cancle">
           <property name="text">
            <string>取消</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
